modules/data-loading/partials/create-data-source.adoc

```diff
@@ -0,0 +1,23 @@
+=== Create data source
+Now that the connector has started loading data into TigerGraph's internal Kafka cluster, you can create a data source and point it to the Kafka cluster:
+
+. Create a data source configuration file.
+The broker's IP and hostname should be `localhost:30002`, which points to the port for TigerGraph's internal Kafka cluster.
```

Comment:
Not necessarily localhost:30002 even for tigergraph kafka. It is only localhost when tigergraph is installed on single node cluster. It is usually the private IP address of the node and it can be retrieved using `gmyip`